tornado/test/netutil_test.py

```diff
@@ -96,6 +96,28 @@ def tearDown(self):
         super(BlockingResolverErrorTest, self).tearDown()
 
 
+@skipIfNoNetwork
+@unittest.skipIf(futures is None, "futures module not present")
+class OverrideResolverTest(AsyncTestCase, _ResolverTestMixin):
+    def setUp(self):
+        super(OverrideResolverTest, self).setUp()
+        mapping = {
+            ('localhost', 80): ('127.0.0.1', 80),
+            ('localhost', 80, socket.AF_INET): ('127.0.0.1', 80),
+            ('localhost', 80, socket.AF_INET6): ('::1', 80)
+        }
+        self.resolver = OverrideResolver(ThreadedResolver(), mapping)
```

Comment:
If you change ThreadedResolver to BlockingResolver, you don't need to skip the test when `futures is None`.

Reply:
Thanks. Will change to ```BlockingResolver```